src/views/studio/MyVideosView/MyVideosView.tsx

```diff
@@ -41,7 +46,7 @@ export const MyVideosView = () => {
         isPublic_eq,
       },
     },
-    { notifyOnNetworkStatusChange: true }
+    { notifyOnNetworkStatusChange: true, nextFetchPolicy: 'cache-and-network' }
```

Comment:
This is needed because if I for example create a public video when I have selected the unlisted tab it will not refetch the video if I go to All videos. 
But just noticed - there is a downside to this - If I create an unlisted video when I have an unlisted tab selected we will not see a thumbnail right away.